reporting/tests/seed.py

```diff
@@ -25,17 +26,16 @@ def create_project():
     project = {
         "name": "Integration Test Suite Project",
         "description": f"Test generated at {ts}",
-        "assets": [],
-        "active": True,
```

Comment:
Did we remove the active tag on projects in the HMI server? Or will it just default?

Reply:
yeah we removed them. We now use `createdOn`, `updatedOn`, and `deletedOn`